paper_framework_sources/p2079_system_execution_context.md

```diff
@@ -104,6 +114,7 @@ public:
    If there is outstanding work at the point of destruction, `std::terminate` will be called.
  - The `system_context` must outlive schedulers obtained from it.
    If there are outstanding schedulers at destruction time, this is undefined behavior.
+ - The lifetime of a `system_context` must end prior to the end of `main()`.
```

Comment:
Do you want to continue down this path, or take the approach of using an implicit context whose lifetime is guaranteed to end at end of main? The feedback last time we discussed it was leaning in that direction.